common/djangoapps/third_party_auth/tests/test_provider.py

```diff
@@ -146,6 +148,16 @@ def test_provider_disabled_for_mismatching_site(self):
         prov = self.configure_google_provider(visible=True, enabled=True, site=site_b)
         self.assertEqual(prov.enabled_for_current_site, False)
 
+    @ddt.data(SITE_DOMAIN_A, SITE_DOMAIN_B)
+    def test_single_provider_for_multiple_sites(self, site_domain):
+        """
+        Verify that enabled_for_current_site returns True when the provider is configured for a multiple sites.
```

Comment:
```python
@ddt.data(SITE_DOMAIN_A, SITE_DOMAIN_B)
def test_single_provider_for_multiple_sites(self, site_domain):
    """
    Verify that "enabled_for_current_site" returns True when the auth provider is configured for multiple sites.
    """
    site, __ = Site.objects.get_or_create(domain=site_domain, name=site_domain)
    auth_provider = self.configure_google_provider(visible=True, enabled=True, site=site)
    with patch('django.contrib.sites.models.SiteManager.get_current', return_value=site):
        assert auth_provider.enabled_for_current_site
```

Reply:
@ziafazal Please replace `test_single_provider_for_multiple_sites` with the above code.